README.md

```diff
@@ -5,10 +5,11 @@ daemon for [OpenShift](https://openshift.io/).
 
 The daemon is meant to be started by the
 [cluster-node-tuning-operator](https://github.com/openshift/cluster-node-tuning-operator).
-The operator supplies configuration data to the daemon by volume-mounting ConfigMaps into 
+The operator supplies configuration data to the daemon by volume-mounting ConfigMaps into
 the container.
 
-The tuned pod uses inotify events to catch ConfigMap changes and
-reloads the profiles based on a newly recommended profile.  Node label
-changes are listened to by using a pull model by querying OpenShift
-API server to fetch node labels.
+The tuned pod:
+  - watches for inotify events to catch profile and the tuned `recommend.conf` ConfigMap changes
+  - watches for node and label changes of pods running on the node the tuned pod runs itself
```

Comment:
node that the tuned pod runs itself.
or
node, which the tuned pod runs itself.

This sentence isn't very clear.

Reply:
Let's say the tuned pod runs on a node _A_.  There are also running other pods on that node (_A_).  The tuned pod cares only about node _A_ labels and labels of pods running on node _A_.  Would the following make it more clear?

- watches for node and pod label changes of all pods running on the node that the tuned pod runs on